languages/python/exercises/concept/dicts/.docs/instructions.md

```diff
@@ -0,0 +1,32 @@
+In this exercise you'll be managing an inventory system.
+
+You will be given a list of items, each time an item is in the given list add `1` to the key in the _given_ inventory. Each item should be organized by their name and the amount of that item. You will also have to delete items from a dictionary.
```

Comment:
Since the issue talks about the student learning how to create a dictionary via constructor and/or literal, maybe we add instructions to that effect?  Something along the lines of _"Given the list of items, create a dictionary to track inventory amounts.  Add/delete items as necessary, and make sure amounts stay current."_